### PR TITLE
Add fw rule's remote group id for create/update

### PR DESCRIPTION
**Pull request by gildub**
We can use source_security_group_id now that we have it in the inventory:

https://github.com/ManageIQ/manageiq-providers-openstack/pull/39

### CODE REVIEW

**Comment by gildub** [comment]
@miq-bot assign @aufi

File: app/models/manageiq/providers/openstack/network_manager/security_group.rb [hunk 1]
```diff
@@ -18,12 +18,15 @@ class ManageIQ::Providers::Openstack::NetworkManager::SecurityGroup < ::Security
   end
 
   def self.parse_security_group_rule(rule)
+    if rule["source_security_group_id"] && !rule["source_security_group_id"].empty?
```
Comment (aufi) [inline app/models/manageiq/providers/openstack/network_manager/security_group.rb]: Nit: Couldn't the condition be written with Rails method ```present?```

Comment (gildub) [inline app/models/manageiq/providers/openstack/network_manager/security_group.rb]: Ah yeah, I've not used it much but I'll. Thanks

**Comment by gildub** [comment]
Needed by https://github.com/ManageIQ/manageiq-ui-classic/pull/1011

**Pull request merged**